src/Parser.js

```diff
@@ -1062,8 +1063,61 @@ export default class Parser {
                     body: arg,
                     star: star,
                 }, "text"), nucleus);
+        } else if (text === "$") {
+            return newDollar(nucleus);
+        }
+        // At this point, we should have a symbol, possibly with accents.
+        // If it's in the symbol table or CJK, we're done.
+        if (symbols[this.mode][text]) {
+            this.consume();
+            return newArgument(new ParseNode(symbols[this.mode][text].group,
+                text, this.mode, nucleus), nucleus);
+        } else if (cjkRegex.test(text)) {
+            this.consume();
+            return newArgument(
+                new ParseNode("textord", text, this.mode, nucleus));
```

Comment:
If we bailing here do we need to include the `combiningDiacriticalMarkString` with surrogate pairs in the regex in Lexer.js?

Reply:
`combiningDiacriticalMarkString` is already included in Lexer's `tokenRegex` both after a single codepoint and after a surrogate pair.

Reply:
We have to lex the tokens before we can test the tokens against the cjkRegex here, d'oh.